### ISSUE TITLE
Google Maps card won't display mobile users' locations

### ISSUE THREAD

**Issue by mkanet** [home-assistant/android#2915]
I know there's a breaking change in 2022.9 for latitude and longitude.  However, I didn't see a respective solution for how to display mobile user locations in the Google Maps card like I had with all previous versions of Home Assistant mobile.  I have temporarily reverted back Home Assistant mobile on all our phones to 2022.8 until I know how to display users' locations in the Google Maps card again.

Could someone please assist me to fix the still image URL below to use respective longitude/latitude of each user?  I currently have the still image URL in configuration.yaml; as, I don't see a place to enter the URL in the Google Maps lovelace card.

Thank you so much for your help!

```
  - platform: generic
    name: 'Google Maps'
    still_image_url: https://maps.googleapis.com/maps/api/staticmap?size=800x800&maptype=hybrid&markers=color:red|label:A|{{ states.sensor.sm_g781v_2_geocoded_location.attributes.latitude }},{{ states.sensor.sm_g781v_2_geocoded_location.attributes.longitude }}&markers=color:purple|label:A|{{ states.sensor.sm_g781v_geocoded_location.attributes.latitude }},{{ states.sensor.sm_g781v_geocoded_location.attributes.longitude }}&markers=color:yellow|label:M|{{ states.device_tracker.kftrwi.attributes.latitude }},{{ states.device_tracker.kftrwi.attributes.longitude }}&markers=color:blue|label:M|{{ states.sensor.sm_n960u_geocoded_location.attributes.latitude }},{{ states.sensor.sm_n960u_geocoded_location.attributes.longitude }}&key=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
    limit_refetch_to_url_change: true

```

**Home Assistant Android version:**
2022.9

**Android version:**
Android 12

**Phone model:**
Samsung Note 

**Home Assistant version:**
2022.9.4

**Last working Home Assistant release (if known):**
2022.8

**Description of problem:**
Google Maps card no longer displays location of Home Assistant mobile users

**Traceback (if applicable, to get the logs you may refer to: https://companion.home-assistant.io/docs/troubleshooting/faqs/#android-crash-logs):**

```

```

**Screenshot of problem:**

**Additional information:**

**Comment by dshokouhi** [comment]
you just need to adjust the templates used, replace `sensor.geocoded_location` with each device

The `latitude` can be retrieved using: `{{ state_attr('sensor.geocoded_location', 'location')[0] }}`

and the `longitude`: `{{ state_attr('sensor.geocoded_location', 'location')[1] }}`

**Comment by mkanet** [comment]
@dshokouhi thank you.  Unfortunately, the Google Maps card is producing a `staticmaperror`.  

**I have triple checked in the template editor** that I have entered the correct syntax for each user's latitude and longitude.  Is there something else I need to do for the Google Maps card to work correctly again?  
![Screenshot 09-24-2022 at 06 58 02 PM](https://user-images.githubusercontent.com/811008/192125055-48b93a1a-1261-462e-8609-a5ae9f7d8b5d.png)

```
still_image_url: https://maps.googleapis.com/maps/api/staticmap?size=800x800&maptype=hybrid&markers=color:red | label:A | {{ state_attr('sensor.sm_g781v_2_geocoded_location', 'location')[0] }}, {{ state_attr('sensor.sm_g781v_2_geocoded_location', 'location')[1] }}&markers=color:yellow | label:M | {{ state_attr('sensor.sm_t500_geocoded_location', 'location')[0] }}, {{ state_attr('sensor.sm_t500_geocoded_location', 'location')[1] }}&markers=color:blue | label:M | {{ state_attr('sensor.sm_n960u_geocoded_location', 'location')[0] }}, {{ state_attr('sensor.sm_n960u_geocoded_location', 'location')[1] }}&key=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```
Is there another breaking change that I don't know about in Home Assistant Core?  Do I need to revert back to an older version of Home Assistant?

**Comment by dshokouhi** [comment]
Your URL looks to have extra spacing. I don't use Google maps card so I can't help much there in the template syntax. Maybe go back to the forums or place you found the card and ask for help there?

**Comment by mkanet** [comment]
Thanks @dshokouhi.  I appreciate your help.  I'm using the syntax provided in the official Google Maps URL below using the same spacing.  I've also tried removing the spacing.  Hopefully someone has to know what broke and how to fix it.

https://www.home-assistant.io/examples/google_maps_card/

**Comment by dshokouhi** [comment]
per the example in the docs (note: I replaced the example attributes with what you should be using)

`https://maps.googleapis.com/maps/api/staticmap?center={{ state_attr('sensor.geocoded_location', 'location')[0] }},{{ state_attr('sensor.geocoded_location', 'location')[1] }}&zoom=13&size=500x500&maptype=roadmap&markers=color:blue%7Clabel:P%7C{{{ state_attr('sensor.geocoded_location', 'location')[0] }},{{ state_attr('sensor.geocoded_location', 'location')[1] }}&key=YOUR_API_KEY`

does not match what you posted above

`https://maps.googleapis.com/maps/api/staticmap?size=800x800&maptype=hybrid&markers=color:red | label:A | {{ state_attr('sensor.sm_g781v_2_geocoded_location', 'location')[0] }}, {{ state_attr('sensor.sm_g781v_2_geocoded_location', 'location')[1] }}&markers=color:yellow | label:M | {{ state_attr('sensor.sm_t500_geocoded_location', 'location')[0] }}, {{ state_attr('sensor.sm_t500_geocoded_location', 'location')[1] }}&markers=color:blue | label:M | {{ state_attr('sensor.sm_n960u_geocoded_location', 'location')[0] }}, {{ state_attr('sensor.sm_n960u_geocoded_location', 'location')[1] }}&key=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx`

**Comment by mkanet** [comment]
Thank you for trying to help.  It looks like there was yet another breaking change in in Home Assistant (or a new bug is introduced).  Reverting back to Home Assistant version 2022.7.7 using what I posted above works as expected.

**Comment by dshokouhi** [comment]
what happens if you render the URL in Developer Tools > Templates and then visit that URL in teh browser? does it work? Sorry thats about as far as I can take it. Hopefully someone helps out.

**Comment by mkanet** [comment]
I see the same image as I do in the card... with an error and no users are displayed.

**Comment by dshokouhi** [comment]
Did you try using the device tracker instead of the geocoded sensor?

**Comment by mkanet** [comment]
Dan your previous comments gave me the clue to resolving this issue!  Thank you!!!

You were right, for whatever reason, the spaces in the URL will cause an error in Google's API.  I tried again your suggestion to render the URL **without spaces** in Developer Tools > Templates and then visit that URL in the browser... it worked in the browser!  That's when I suspected the issue would go away if I just remove the spaces from the URL in the configuration.yaml file.

I really appreciate your help.  You're a life-saver!
